files/setup_bastion.yaml

```diff
@@ -42,6 +42,17 @@
     net2Routes: __net2_routes__
     extraGateway: __extra_gateway__
   tasks:
+
+    - name: Check if stack update or create and register variable
+      shell: if [[ $(openstack stack show "openshift-{{ osTenantName }}" -f value -c stack_status \
+                --os-auth-url "{{ osAuthUrl }}" \
+                --os-username "{{ openstackOpenshiftUsername }}" \
+                --os-project-name "{{ osTenantName }}" \
+                --os-password "{{ openstackOpenshiftPassword }}") = *CREATE* ]] ; then echo "True" ; else echo "False" ; fi
+      register: create_value
+
+    - set_fact: create="{{ create_value.stdout }}"
```

Comment:
Can we come up with a less ambiguous variable name than "create" ? e.g. is_stack_create

Reply:
Changed create to stack_create